.dockerignore

```diff
@@ -1,7 +1,4 @@
-.env
-.env.example
-composer.json
-composer.lock
```

Comment:
We need them:
* .env.* to check if Dotenv library works in Docker (once the wrapper to fetch either from PHP getenv or from Dotenv is ready, we can drop them)
* composer.* to install packages in the container